cli/src/lib.rs

```diff
@@ -323,6 +323,65 @@ pub fn post_share(
     ))
 }
 
+pub fn create_verify_token(
+    host: &str,
+    id: uno::Id,
+    email: Option<String>,
+) -> Result<String>
+{
+    let keypair = uno::KeyPair::from(id);
+    let url = verify_token_url_from_public_key(host, &keypair.public)?;
+
+    #[derive(Serialize)]
+    struct VerifyCreateBody
+    {
+        email: Option<String>,
+    }
+
+    let json =
+        serde_json::to_string(&VerifyCreateBody { email: email.clone() })?;
+
+    let req = surf::post(url.as_str()).body(json).build();
+
+    async_std::task::block_on(do_http_signed(req, &id))
+        .map_err(|e| anyhow!("{}", e))?;
+
+    Ok(format!(
+        "verify token created at {} for email: {}",
+        url,
+        email.unwrap_or("(empty)".to_string())
+    ))
+}
+
+pub fn verify_verify_token(
+    host: &str,
+    id: uno::Id,
+    secret: String,
+) -> Result<String>
+{
+    let keypair = uno::KeyPair::from(id);
+    let url = verify_token_url_from_public_key(host, &keypair.public)?;
+
+    #[derive(Serialize)]
+    struct VerifyVerifyBody
+    {
+        secret: String,
+        method: uno::VerifyMethod,
+    }
+
+    let json = serde_json::to_string(&VerifyVerifyBody {
+        secret: secret,
+        method: uno::VerifyMethod::Cli,
```

Comment:
this would still be email since the only way to view the token is by looking in your email. the fact that you used the cli to complete the request is not particularly important.